doc/articles/features/status-bar-theme-color.md

```diff
@@ -0,0 +1,148 @@
+
+The [`UISettings.ColorValuesChanged` event](https://docs.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.uisettings.colorvalueschanged?view=winrt-19041) can be used to listen when the dark mode is turned on or off.
+
+## Example
+The complete sample code can be found here: [StatusBarThemeColor](https://github.com/unoplatform/Uno.Samples/tree/master/UI/StatusBarThemeColor)
+Or, you can follow along the step-by-step instructions:
+
+1. Create a new uno app: [get started](..\get-started.md)
+2. In `MainPage.xaml`, add a `<CommandBar>`:
+    > On iOS, the status bar color cannot be set directly, so it is done via through a CommandBar placed in the page. You could also use any xaml element like `<Grid>` or `<Border>` to achieve similar effect, if your application doesnt use any navigation or doesn't use native navigation. This is because the page content can goes under the status bar. In fact, you usually have to add padding to avoid that (see next step).
+    ```xml
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <CommandBar Grid.Row="0"
+                    x:Name="MyCommandBar"
+                    Content="MainPage" />
+
+        <StackPanel Grid.Row="1">
+            <TextBlock Text="Hello, world!"
+                        Margin="20"
+                        FontSize="30" />
+        </StackPanel>
+    </Grid>
+    ```
+3. Add a `VisibleBoundsPadding.PaddingMask` the root `<Grid>` to prevent the content being stuck under status & command bar:
+    ```xml
+    <Page ...
+          xmlns:toolkit="using:Uno.UI.Toolkit"
+          ...>
+
+        <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+              toolkit:VisibleBoundsPadding.PaddingMask="Top">
+    ```
+4. In `MainPage.xaml.cs`, expose the `MyCommandBar` which will be referenced in later step:
+    ```cs
+    public static MainPage Instance { get; private set; }
+
+    public MainPage()
+    {
+        Instance = this;
+
+        this.InitializeComponent();
+    }
+
+    public CommandBar GetCommandBar() => MyCommandBar;
+    ```
+
+5. In `App.xaml.cs`, enable the native frame navigation to use the native style for `CommandBar`, instead of the UWP style:
+    ```cs
+    public App()
+    {
+        ConfigureFilters(global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory);
+
+    #if __IOS__ || __ANDROID__
+        // Enable native frame navigation.
+        Uno.UI.FeatureConfiguration.Style.ConfigureNativeFrameNavigation();
+    #endif
+
+        this.InitializeComponent();
+        this.Suspending += OnSuspending;
+    }
+    ```
+
+6. Subscribe to the `UISettings.ColorValuesChanged` event from `App.xaml.cs`:
+    > Note that the instance of `UISettings` is kept to prevent it from disposed when going out of scope, which will dispose the event subscription.
```

Comment:
```suggestion
    > Note that the instance of `UISettings` is kept to prevent it from disposed when going out of scope, which would otherwise dispose the event subscription.
```